src/Moq/Linq/MockSetupsBuilder.cs

```diff
@@ -116,33 +116,10 @@ private static Expression ConvertToSetup(Expression left, Expression right)
 		{
 			switch (left.NodeType)
 			{
-				case ExpressionType.MemberAccess:
-					var member = (MemberExpression)left;
-					Guard.NotField(member);
```

Comment:
This check are removed because `expression.Split` and `InvocationShape` should already perform their own checks.